lstchain/io/io.py

```diff
@@ -1018,7 +1025,46 @@ def read_dl2_to_pyirf(filename):
     events = pd.read_hdf(filename, key=dl2_params_lstcam_key).rename(columns=name_mapping)
     events = table.QTable.from_pandas(events)
 
+    #Make the columns as Quantity
     for k, v in unit_mapping.items():
         events[k] *= v
 
-    return events, pyirf_simu_info
\ No newline at end of file
+    return events, pyirf_simu_info
+
+def read_data_dl2_to_QTable(filename):
+    """
+    Read data DL2 files from lstchain and return QTable format
+
+    Parameters
+    ----------
+    filename: path to the lstchain DL2 file
+    Returns
+    -------
+    `astropy.table.QTable`
+    """
+    #Mapping
+    name_mapping = {
+        'gammaness' : 'gh_score',
+        'alt_tel': 'pointing_alt',
+        'az_tel': 'pointing_az',
+
+    }
+    unit_mapping = {
+        'reco_energy': u.TeV,
+        'pointing_alt': u.rad,
+        'pointing_az': u.rad,
+        'reco_alt': u.rad,
+        'reco_az': u.rad,
+		'dragon_time': u.s,
```

Comment:
Indention here is mixed tabs/spaces. This should not happen with any good editor.

Python should be indented only by 4 spaces / level.

Reply:
Corrected. Thanks!